### PR TITLE
add missing snapshot yaml examples

### LINKED ISSUE #6382: Add updated YAML snapshot configs in the general config pages
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Some of the resources listed under the general config pages don't have the updated 1.9+ YAML configs listed or mentioned. We should update the snapshot tabs to show the YAML configuration.

```yaml
snapshots:
  - name: <string>
    config:
      [database](/reference/resource-configs/database): <string>
      [schema](/reference/resource-configs/schema): <string>
      [unique_key](/reference/resource-configs/unique_key): <column_name_or_expression>
      [strategy](/reference/resource-configs/strategy): timestamp | check
      [updated_at](/reference/resource-configs/updated_at): <column_name>
      [check_cols](/reference/resource-configs/check_cols): [<column_name>] | all
      [snapshot_meta_column_names](/reference/resource-configs/snapshot_meta_column_names): {<dictionary>}
```

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/reference/resource-configs/enabled

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
Some of the resources listed under the general config pages don't have the updated 1.9+ YAML configs listed or mentioned. 

This PR updates the snapshot tabs to show the YAML configuration for the resources missing it.

Resolves #6382

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/alias
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/contract
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/database
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/enabled
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/group
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/persist_docs
- https://docs-getdbt-com-git-add-snapshot-yaml-dbt-labs.vercel.app/reference/resource-configs/schema

### CODE REVIEW

File: website/docs/reference/resource-configs/enabled.md [hunk 1]
```diff
@@ -78,6 +78,19 @@ snapshots:
 
 </File>
 
+<File name='snapshots/properties.yml'>
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/enabled.md]: Should we version this for 1.9 as well?

File: website/docs/reference/resource-configs/schema.md [hunk 2]
```diff
@@ -22,6 +22,23 @@ models:
 ```
 </File>
 
+In a `snapshots/snapshot_name.yml` file:
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/schema.md]: This sentence doesn't flow from the previous sentence (`For example, if you have a group of marketing-related models and want to place them in a separate schema called `marketing`, you can configure it like this:
`) should it or should it be its own sentence?



Also, should this sentence be included in the version block?

**Review by runleonarun — CHANGES_REQUESTED** [review]
@mirnawong1 I just had some minor questions

File: website/docs/reference/resource-configs/enabled.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/reference/resource-configs/enabled.md]: yes good call, thank you!

File: website/docs/reference/resource-configs/schema.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/reference/resource-configs/schema.md]: oh jeez, this doesn't even belong in this tab! that's my miss - thank you for catching this

Comment (mirnawong1) [reply website/docs/reference/resource-configs/schema.md]: i need to move this to the snapshots tab

File: website/docs/reference/resource-configs/alias.md [hunk 3]
```diff
@@ -117,15 +117,15 @@ snapshots:
...
 
 ```yml
 version: 2
 
 snapshots:
-  - name: your_snapshot
+  - name: your_snapshot_name
     config:
       alias: the_best_snapshot
 ```
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/alias.md]: Pushign commit to restart build
```suggestion
```

```

**Review by runleonarun — APPROVED** [review]
Looks great!

**Pull request merged**